packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -589,11 +592,17 @@ class Tooltip extends OverlayClassMixin(ThemePropertyMixin(ElementMixin(PolymerE
   /** @private */
   __effectiveAriaTargetChanged(ariaTarget, oldAriaTarget) {
     if (oldAriaTarget) {
-      removeValueFromAttribute(oldAriaTarget, 'aria-describedby', this._uniqueId);
+      const targets = Array.isArray(oldAriaTarget) ? oldAriaTarget : [oldAriaTarget];
+      targets.forEach((target) => {
+        removeValueFromAttribute(target, 'aria-describedby', this._uniqueId);
+      });
     }
 
     if (ariaTarget) {
-      addValueToAttribute(ariaTarget, 'aria-describedby', this._uniqueId);
+      const targets = Array.isArray(ariaTarget) ? ariaTarget : [ariaTarget];
+      targets.forEach((target) => {
```

Comment:
nit: could be shortened to `[ariaTarget].flat().forEach(...)`

Reply:
Thanks, done.